code/app/build.gradle

```diff
@@ -14,15 +16,16 @@ spotless {
 }
 
 android {
-    compileSdkVersion 30
-    buildToolsVersion "30.0.2"
+      namespace "com.adobe.marketing.mobile.consentTestApp"
+      compileSdk rootProject.ext.compileSdkVersion
+      buildToolsVersion rootProject.ext.buildToolsVersion
```

Comment:
buildToolsVersion should no longer be required with latest versions of agp - https://developer.android.com/studio/releases/gradle-plugin.html#behavior_changes_1